releases/release-7.1.0.md

```diff
@@ -23,35 +25,71 @@ In v7.1.0, the key new features and improvements are as follows:
 </thead>
 <tbody>
   <tr>
-    <td rowspan="2">Scalability and Performance</td>
-    <td><a href="https://docs.pingcap.com/tidb/dev/sql-non-prepared-plan-cache" target="_blank">Session-level plan cache for non-prepared statements</a> (GA)</td>
-    <td>Support automatically reusing plan cache at the session level to remove query planning time, reducing query time for repeat SQL patterns without manually setting prepare statements in advance.</td>
+    <td rowspan="3">Scalability and Performance</td>
+    <td>TiFlash supports the <a href="https://docs.pingcap.com/tidb/dev/tiflash-disaggregated-and-s3" target="_blank">disaggregated storage and compute architecture and S3 shared storage</a> (experimental, introduced in v7.0.0)</td>
+    <td>TiFlash introduces a cloud-native architecture as an option:
+      <ul>
+        <li>Disaggregates TiFlash's compute and storage, which is a milestone for elastic HTAP resource utilization.</li>
+        <li>Introduces S3-based storage engine, which can provide shared storage at a lower cost.</li>
+      </ul>
+    </td>
+  </tr>
+  <tr>
+    <td>TiKV supports <a href="https://docs.pingcap.com/tidb/dev/system-variables#tidb_store_batch_size" target="_blank">batch aggregating data requests</a> (introduced in v6.6.0) </td>
+    <td>This enhancement significantly reduces total RPCs in TiKV batch-get operations. In situations where data is highly dispersed and the gRPC thread pool has insufficient resources, batching coprocessor requests can improve performance by more than 50%.</td>
   </tr>
   <tr>
     <td><a href="https://docs.pingcap.com/tidb/dev/troubleshoot-hot-spot-issues#scatter-read-hotspots" target="_blank">Load-based replica read</a></td>
     <td>In a read hotspot scenario, TiDB can redirect read requests for a hotspot TiKV node to its replicas. This feature efficiently scatters read hotspots and optimizes the use of cluster resources. To control the threshold for triggering load-based replica read, you can adjust the system variable <a href="https://docs.pingcap.com/tidb/dev/system-variables#tidb_load_based_replica_read_threshold-new-in-v700" target="_blank"><code>tidb_load_based_replica_read_threshold</code></a>.</td>
   </tr>
```

Comment:
```suggestion
  </tr>
  <tr>
      <td>TiKV supports<a href="https://docs.pingcap.com/tidb/dev/partitioned-raft-kv" target="_blank"> partitioned Raft KV storage engine </a> (experimental)</td>
    <td>TiKV introduces the next-generation storage engine, the partitioned Raft KV. By allowing each data region to have a dedicated RocksDB instance, it can expand the cluster's storage capacity from TB-level to PB-level and provide more stable write latency and stronger scalability.</td>
    </tr>
```